src/fmu/tools/sensitivities/_designsummary.py

```diff
@@ -34,7 +40,20 @@ def summarize_design(filename, sheetname="DesignSheet01"):
 
     """
 
-    # Initialisation of dataframe to store results
+    # Read design matrix
+    if str(filename).endswith(".xlsx"):
+        dgn = pd.read_excel(filename, sheetname, engine="openpyxl")
```

Comment:
I'm sure that you don't want to change the naming conventions but `dgn` is not a great name to be honest.